src/DynamoCore/Migration/Migration.cs

```diff
@@ -251,6 +251,13 @@ orderby result.From
                 string typeName = elNode.Attributes["type"].Value;
                 typeName = Nodes.Utilities.PreprocessTypeName(typeName);
 
+                // If we try to migrate Function node, then use its' nickname as type.
+                if (typeName == "Dynamo.Nodes.DSFunction")
```

Comment:
I couldn't fing any example of DSFunction. I.e. when we try to migrate function node. Seems it's the first time :)